modules/ROOT/pages/amqp/amqp-connector.adoc

```diff
@@ -62,9 +62,9 @@ Anypoint Exchange and click *Dependency Snippets*.
 
 === Define Global Defaults
 
-The AMQP `config` allows you to define multiple parameters that will be used by default while consuming or publishing messages. This way, you can define a global default behavior for all the operations associated to the config and then override each parameter only in the operations that require a custom behavior.
+The AMQP `config` allows you to define multiple parameters to use by default while consuming or publishing messages. This way, you can define a global default behavior for all the operations associated with the config and then override each parameter only in the operations that require a custom behavior.
```

Comment:
What should we call `config`? Is it a command? a value? some other thing?
"The... `config` <option> enables you to...."

Also, we later refer to "config" without the syntax formatting. In that case, should we be spelling out "configuration" because that's what we mean: the thing rather than the syntax?

Reply:
I had to think about the meaning of this for a while, and I think I got tripped up by the seeming contradiction between "multiple" and "default" (a default typically implying one preferred "thing"). 
I could very well be overthinking this, but I wonder if we should call this "a set of default parameters"?